dev.js

```diff
@@ -89,10 +89,10 @@ module.exports = {
     /**
      * Function to create an account on local node. This will not work on non-dev environments.
      */
-    createAccountWithLocalNodeConnection: async function (newAccountName, newAccountPublicKey) {
+    createAccountWithLocalNodeConnection: async function (newAccountName, newAccountPublicKey, amount = 1000000000000) {
```

Comment:
If we add additional parameter here it also needs to be introduced for `createAccountWithContractHelper` which would require plumbing in contract helper. Probably easier to use constant, one always can use lower level API for more control.

Reply:
done